index.php

```diff
@@ -314,7 +314,8 @@ public function submit_request() {
 			$headers  = 'MIME-Version: 1.0' . "\r\n";
 			$headers .= 'Content-type: text/html; charset=iso-8859-1' . "\r\n";
 			$headers .= 'From:' . $name . '<' . $from . '>' . "\r\n";
-			echo mail( $to, $subject, $html, $headers ) ? esc_html_e( 'Thank you!', 'all-in-one-schemaorg-rich-snippets' ) : esc_html_e( 'Something went wrong!', 'all-in-one-schemaorg-rich-snippets' );
+			$result  .= wp_mail( $to, $subject, wp_kses_post( $html ), $headers );
```

Comment:
Remove . from this line

it should be $result = when defining for first time

Reply:
updated @sushmak02